dev/devicelab/lib/framework/devices.dart

```diff
@@ -62,6 +62,13 @@ enum DeviceOperatingSystem {
   windows,
 }
 
+extension Darwin on DeviceOperatingSystem {
```

Comment:
extension methods are against Flutter's style guide https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-extension

And I believe you can achieve the same effect now with Dart's new enhanced enum's: https://dart.dev/guides/language/language-tour#enumerated-types https://medium.com/dartlang/dart-2-17-b216bfc80c5d